biome.jsonc

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.7.3/schema.json",
+  "organizeImports": {
+    "enabled": true
+  },
+
+  "formatter": {
+    "enabled": true,
+    "indentStyle": "space"
+  },
+  "javascript": {
+    "formatter": {
+      "trailingComma": "es5"
+    }
+  },
+  "linter": {
+    "enabled": true,
+    "rules": {
+      "recommended": true,
+      "suspicious": {
+        "noExplicitAny": "off"
```

Comment:
This is a good rule to keep enabled. There are definitely some situations where `any` is the correct type to use, but they should be pretty rare, and the exceptions clearly explained when done.

Reply:
Sorry, but it gets onerous to keep casting `unknowns` in `_parse` implementations and tests. It's pretty common to use `any` in library code.